packages/server/src/project/project.decorator.ts

```diff
@@ -0,0 +1,10 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import jwt_decode from 'jwt-decode';
+
+export const GetProject = createParamDecorator((data: unknown, ctx: ExecutionContext) => {
+  const request = ctx.switchToHttp().getRequest();
+  const token = request.headers.authorization.split(' ')[1];
```

Comment:
```suggestion
  const token = req.headers?.authorization?.split(' ')[1];
```

Possibly no authorization or not space in header, have to handle that case